neptune/DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: neptune
 Type: Package
 Title: Interface to 'Neptune' Experiment Tracking and Management Tool
-Version: 0.1.1
+Version: 0.1.2
 Authors@R: c(
```

Comment:
I talked with Marcin and Artifacts API should be supported as well. Check these: https://docs.neptune.ai/api-reference/field-types#artifact

Reply:
Especially check the `track_files` operation

Reply:
In other words. I couldn't find a matching function instead of directly call it via `run['a']$track_files(...)`